lib/deckard/build.ex

```diff
@@ -21,7 +21,8 @@ defmodule Deckard.Build do
             version: data["distro_version"],
             sha_sum: data["sha_sum"],
             size: String.to_integer(data["byte_size"]),
-            url: build_url(data["path"])
+            url: build_url(data["path"]),
+            urgent: Keyword.get(data, "urgent", false)
```

Comment:
Do we want `Map` here instead? Based on the other usage it seems this is a map.

Reply:
Whoops, I read line 15 wrong. You're right